.travis.yml

```diff
@@ -4,8 +4,15 @@ jdk:
   - openjdk8
   - openjdk11
 addons:
-  postgresql: "9.4"
-  firefox: "60.3.0esr"
+  postgresql: "11"
+  apt:
+    packages:
+      - postgresql-11
+      - postgresql-client-11
+  firefox: "68.2.0esr"
+env:
+  global:
+  - PGPORT=5432
```

Comment:
- [ ] Isn't the default value of `PGPORT` 5432?

`PGPORT` is client (e.g. `createdb`) default settings.
https://www.postgresql.jp/document/8.2/html/libpq-envars.html

Reply:
The commit 37a74f3ef7f5db0c269ccc9c88fca310fe59193b will resolve it.